dll/win32/netcfgx/lang/ja-JP.rc

```diff
@@ -6,17 +6,17 @@ CAPTION "全般"
 FONT 9, "MS UI Gothic"
 BEGIN
     LTEXT "ネットワークでこの機能がサポートされている場合は、IP 設定を自動的に取得することができます。サポートされていない場合は、ネットワーク管理者に適切な IP 設定を問い合わせてください。", -1, 9, 9, 235, 27
-    CONTROL "IP アドレスを自動的に取得する", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP, 14, 43, 210, 12
     GROUPBOX "", -1, 9, 61, 228, 60, BS_GROUPBOX
+    CONTROL "IP アドレスを自動的に取得する", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 43, 210, 12
     CONTROL "次の IP アドレスを使う(&U):", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 59, 120, 12
     LTEXT "IP アドレス:", -1, 14, 75, 135, 8
     CONTROL "", IDC_IPADDR, "SysIPAddress32", WS_TABSTOP, 144, 75 - 2, 85, 12
     LTEXT "サブネット マスク:", -1, 14, 90, 135, 8
     CONTROL "", IDC_SUBNETMASK, "SysIPAddress32", WS_TABSTOP, 144, 90 - 2, 85, 12
     LTEXT "デフォルト ゲートウェイ:", -1, 14, 105, 135, 8
     CONTROL "", IDC_DEFGATEWAY, "SysIPAddress32", WS_TABSTOP, 144, 105 - 2, 85, 12
-    CONTROL "DNS サーバーのアドレスを自動的に取得する", IDC_AUTODNS, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP, 14, 129, 210, 12
     GROUPBOX "", -1, 9, 147, 228, 47, BS_GROUPBOX
+    CONTROL "DNS サーバーのアドレスを自動的に取得する", IDC_AUTODNS, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 129, 210, 12
```

Comment:
@katahiromz in case this line would end with an unintended dot for ja-JP.rc or any of the 3 chinese languages, then please let me know. I am not so familiar with their characters. And that line should never end with a dot.

Reply:
![image](https://github.com/reactos/reactos/assets/2107452/8946bf2a-656a-46c9-a790-8e609935b8fb)
![image](https://github.com/reactos/reactos/assets/2107452/34eecf52-8884-4065-b234-ec442ade50e1)
![image](https://github.com/reactos/reactos/assets/2107452/2dd93c14-6df0-486a-abad-53fc70ddab69)
![image](https://github.com/reactos/reactos/assets/2107452/0c194e75-dcaa-4432-9fd9-772f8df02929)
I cannot found any problems.

Reply:
Thx for your checking. For me it seems as if both radiobuttons which do end with the (U) accelerator should actually end with "(U) :". In those screenshot only the upper one has the ":", but the lower one lacks it. At least both do have the ":" in my german XPSP3 netcfgx.. But I do just mention that here as a fact. I didn't touch that line in my PR, so a fix for that won't be part of my PR neither. I will still leave that discussion-thread in unresolved state.

Reply:
Oh, you mean "colon"?
I think it should be added to both.

Reply:
But Chinese colon is in fullwidth. Be care.

Reply:
Initially I really meant unintended dots, like I stripped it here https://github.com/reactos/reactos/pull/6049/commits/e1173341c3742d5c6144045e8a4e85131cd01546 for example.

The thing with the colon I didn't realize before I saw your screenshot (different thing, and different line in the rc).